run-pty.json

```
@@ -7,6 +7,6 @@
     "defaultStatus": ["⏳", "S"]
   },
   {
-    "command": ["yarn", "build:css", "--watch"]
+    "command": ["npm", "run", "build:css", "--watch"]
```

```suggestion
    "command": ["npm", "run", "build:css", "--", "--watch"]
```

Otherwise it's interpreted as an argument to `npm`